openmls/src/treesync/node/leaf_node.rs

```diff
@@ -718,12 +747,15 @@ impl OpenMlsLeafNode {
         leaf_node_source: LeafNodeSource,
         backend: &impl OpenMlsCryptoProvider,
         credential_bundle: &CredentialBundle,
+        capabilities: Capabilities,
```

Comment:
Looks like comments need to be updated here on the function (and maybe elswhere).